src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
```

Comment:
1. The question about cache provider remains open, we will decide today or when you have time.
2. Using laravel cache with it's settings, in my point of view, brings magic and unexpected results (e.g. reserved cache keys for telegram-bot-sdk) for developers. 
3. What would you say about this order of implementation of cache drivers support: Filesystem -> SQLite3 -> Redis -> Memcached?